File: Numbers/ApSeries.java
public class ApSeries {
    public static double APcheck(double a, double d, int n) {
        double first = a, differnce = d;
        int NumOfTerms = n;
        double sum = 0.0;
        for (int i = 1; i <= n; i++) {
            sum += first;
            first += differnce;

        }
        return sum;
    }

    public static void main(String[] args) {
        double a = 12;
        double d = 10;
        int n = 5;
        System.out.println("The sum of the AP series:" + APcheck(a, d, n));

    }
}

File: Problem Solving Python/Numbers/AP_Series.py
def APSeries(a,d,n):
    first_term=a
    difference=d
    no_terms=n
    sum=0
    for i in range(1,n+1):
        sum += first_term
        first_term += difference
    return sum

if __name__=='__main__':
    a=float(input("Enter the first term of the AP series: "))
    d=float(input("Enter the common difference of the AP series: "))
    n=int(input("Enter the number of terms of AP series: "))
    print("The sum of the AP series is: ",APSeries(a,d,n))